### PR TITLE
Fix/completions

### PR DESCRIPTION
**Pull request by dannil76**
## Explain the changes you’ve made
Updated status text to complete the completions work flow

## Tested environments

- [x] Your personal AWS environment.

---

## Checklist for reviewers

Here are a few points that are useful for reviewers to check.

The short list of the (maybe) most important points are:

- Does this PR work as expected?
- Does the PR properly only concern a single feature/bug/thing?
- Is the code clear and concise?
- Will the changes impact existing functionality?
- Are there any linting errors (ESLint) that should be resolved?
- Do the automatic GitHub checks pass?

However it is good if a reviewer can be as thorough as possible,
within reasonable limits of course.

### Functionality

- Does this PR work as expected?
- If this is a new feature, does it add value, or is it a sign of feature-creep?
- Will the changes impact existing functionality?
- Does the changes create inconsistencies somewhere else?
- Does the code work on different platforms if applicable?
- How does the code handle if unexpected values are passed to functions etc.?

### Readability, syntax, formatting, style

- Is the code clear and concise?
- Has styling rules (Prettier) been applied to included files?
- Are functions/variables/etc. aptly named?

### Design

- Is the code properly planned and designed?
- Is code organized/separated in a logical fashion?
- Does the code follow clean code practices to a reasonable degree?
- Is the code self-documenting enough?
- Is the code in sync with existing patterns/technologies?
- Does the code reuse existing functionality where applicable?
- Is the code designed with reusability in mind?
- Does the code have hard-coded values that can be refactored?

### Patterns, idioms, tests, best practices

- Are there any linting errors (ESLint) that should be resolved?
- Is the code properly typed (TypeScript)?
- Does the code utilize language features to their full potential?

### Debugging, testing, configuring

- Is appropriate logging used?
- Have tests been added/modified to accommodate the changes?
- Is the feature sufficiently configurable in case tweaks are required in the future?

### Performance, reliability, scalability

- Is the code well optimized?
- Does the code scale?
- Does the code handle failure scenarios?

### Security

- Is there any obvious potential for exploits and security threats?
- Does code check proper authorization where applicable?

### Etiquettes

- Are commits atomic (does only one thing per commit)?
- Does the PR properly only concern a single feature/bug/thing?
- Are commit messages well-written?
- Do the automatic GitHub checks pass?

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-2qa1anj Uppdatera statustexter som berör completions](https://app.clickup.com/t/2qa1anj)

File: libs/caseStatuses.js [hunk 1]
```diff
@@ -50,13 +52,13 @@ const statuses = [
...
     name: 'Signerad',
-    description: 'Ansökan är signerad',
+    description: 'Ansökan är signerad.',
   },
   {
     type: ACTIVE_SIGNATURE_PENDING,
     name: 'Väntar på signering',
     description:
-      'Väntar på signering. När båda har signerat med BankID skickas ansökan in för bedömning',
+      'Väntar på signering. När båda har signerat med BankID skickas ansökan in för bedömning.',
```
Comment (kajm) [inline libs/caseStatuses.js:61]: remove `.`

File: libs/caseStatuses.js [hunk 2]
```diff
@@ -111,20 +113,37 @@ const statuses = [
...
+      'Du har skickat in #ATTACHMENT_UPLOADED_COUNT bilder/filer för komplettering.\n\nDu kan skicka in fler fram till: #COMPLETION_DUEDATE',
+    detailedDescription:
+      'Du har kompletterat med #ATTACHMENT_UPLOADED_COUNT bilder/filer. Har du fler bilder att skicka in startar du formuläret och laddar upp kompletterande bilder och signerar med BankID.\n\nDu kan fortsätta skicka in bilder fram till att senaste datumet för inskickning har passerat.\n\nDu kan skicka in fler fram till: #COMPLETION_DUEDATE',
   },
   {
     type: ACTIVE_RANDOM_CHECK_REQUIRED_VIVA,
     name: 'Stickprovskontroll',
     description:
-      'Du måste komplettera din ansökan med bilder som visar dina utgifter och inkomster. Vi behöver din komplettering inom 4 dagar för att kunna betala ut pengar för perioden.',
+      'Du måste komplettera din ansökan med bilder som visar dina utgifter och inkomster. Vi behöver din komplettering inom 3 dagar för att kunna betala ut pengar för perioden.',
```
Comment (kajm) [inline libs/caseStatuses.js:132]: remove `.`

File: libs/caseStatuses.js [hunk 3]
```diff
@@ -111,20 +113,37 @@ const statuses = [
...
+      'Du har skickat in #ATTACHMENT_UPLOADED_COUNT bilder/filer för stickprovskontrollen.\n\nDu kan skicka in fler fram till: #COMPLETION_DUEDATE',
+    detailedDescription:
+      'Du har skickat in #ATTACHMENT_UPLOADED_COUNT för stickprovskontrollen. Har du fler bilder att skicka in startar du formuläret och laddar upp kompletterande bilder på rätt sida och signerar med BankID.\n\nDu kan fortsätta skicka in bilder fram till att senaste datumet för inskickning har passerat.\n\nDu kan skicka in fler fram till: #COMPLETION_DUEDATE',
+  },
+  {
+    type: ACTIVE_PROCESSING_COMPLETIONS_DUE_DATE_PASSED_VIVA,
+    name: 'Ansökan behandlas',
     description:
-      'Stickprovskontroll är inskickad. Du kan fortsätta att skicka in fler underlag till och med förfallodagen för stickprovskontrollen',
+      'Tiden för att lämna in bilder har gått ut. Din komplettering är mottagen och handläggs.',
```
Comment (kajm) [inline libs/caseStatuses.js:146]: remove `.`

**Review by kajm — CHANGES_REQUESTED** [review]

File: libs/caseStatuses.js [hunk 4]
```diff
@@ -50,13 +52,13 @@ const statuses = [
   {
     type: ACTIVE_SIGNATURE_COMPLETED,
     name: 'Signerad',
-    description: 'Ansökan är signerad',
+    description: 'Ansökan är signerad.',
```
Comment (MasterKenth) [inline libs/caseStatuses.js:55]: remove `.`

**Review by MasterKenth — CHANGES_REQUESTED** [review]

**Review by kajm — APPROVED** [review]

**Pull request merged**